src/components/svg-canvas-graph.tsx

```diff
@@ -420,6 +464,26 @@ const SvgCanvas = () => {
                         strokeWidth={svgViewBoxZoom / 100}
                     />
                 ))}
+            {activeSnapPoints && (
+                <>
+                    <line
+                        x1={activeSnapPoints.originalNodesPos[0].x}
+                        y1={activeSnapPoints.originalNodesPos[0].y}
+                        x2={activeSnapPoints.x}
+                        y2={activeSnapPoints.y}
+                        stroke="yellow"
```

Comment:
These two snap points lines are neither visually striking nor effectively comprehensible in their messaging. How can one tell the difference just from color?

Reply:
What do you have in mind? I don't really have a plan.

Reply:
Let's get some suggestion from slack :)